tests/framework/extensions/users/users.go

```diff
@@ -313,7 +313,7 @@ func waitForRTBRollout(client *rancher.Client, rtbNamespace string, rtbName stri
 		Duration: 500 * time.Millisecond,
 		Factor:   1,
 		Jitter:   0,
-		Steps:    11,
+		Steps:    41,
```

Comment:
The comment above can be updated. It currently says:

> we expect rollout to happen within 5 seconds total

but now we're waiting for ~20 seconds.

Reply:
Fixed!